src/main/msp/msp.c

```diff
@@ -2897,8 +2905,15 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, uint8_t cmdMSP,
             sbufReadU8(src);
 #endif
         }
+        break;
 
+    case MSP_SET_RXRANGE_CONFIG:
+        for (int i = 0; i < NON_AUX_CHANNEL_COUNT; i++) {
```

Comment:
Add a preceding count supplied by the client to indicate how many pair values to read. Again to support cases where `NON_AUX_CHANNEL_COUNT` might change. Appropriate handling will also be needed in the Configurator changes so that it bases it's read/write and interface based on the count supplied from the firmware.